src/modules/Profile.rb

```diff
@@ -13,26 +13,10 @@ class ProfileClass < Module
     # All these sections are handled by AutoYaST (or Installer) itself,
     # it doesn't use any external AutoYaST client for them
     GENERIC_PROFILE_SECTIONS = [
```

Comment:
Yes, it it not defined in the desktop file. But has the autoyast desktop file to be installed at all ? I am getting more and more confused....